moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1235,15 +1235,17 @@ void MotionPlanningDisplay::updateStateExceptModified(moveit::core::RobotState&
 void MotionPlanningDisplay::updateQueryStates(const moveit::core::RobotState& current_state)
 {
   std::string group = planning_group_property_->getStdString();
+  if (group.empty())
+    return;
```

Comment:
May want to print a warning here

Reply:
That's not a function triggered by the user. Thus, I don't see the need for a warning here.